server/model/board.go

```diff
@@ -79,6 +84,54 @@ type Board struct {
 	DeleteAt int64 `json:"deleteAt"`
 }
 
+// BoardPatch is a patch for modify boards
+// swagger:model
+type BoardPatch struct {
+	// The type of the board
+	// required: false
+	Type *BoardType `json:"type"`
+
+	// The title of the board
+	// required: false
+	Title *string `json:"title"`
```

Comment:
Why are these pointers to strings, whereas they are just strings in Blocks struct?

Reply:
We need to distinguish on the `(BoardPatch) Patch` function if the user sent an empty title (to set it) or nothing, and then leave the title intact